module/metrics/execution.go

```diff
@@ -22,6 +22,7 @@ type ExecutionCollector struct {
 	lastFinalizedExecutedBlockHeightGauge   prometheus.Gauge
 	stateStorageDiskTotal                   prometheus.Gauge
 	storageStateCommitment                  prometheus.Gauge
+	checkpointSize                          prometheus.Gauge
```

Comment:
Does this need an initialisation?

Reply:
yes, I think it will panic if it's not initialized